CAN-664

vikasranatalentelgia opened the pull request:
Add Support: Implement screen design for Add Support

vikasranatalentelgia committed:
CAN-664

Add Support: Implement screen design for Add Support

akuks commented on src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx:
```diff
@@ -17,12 +17,52 @@ const ManageSupportUI = () => {
         }
       >
+        <span className={styles.warning}>
+          <strong> Warning! </strong>`Agreement` is a parent camp to the list of
```
It should be from the common message file

vikasranatalentelgia replied:
It is a static design implementation for now. will be dynamic once API will be implemented.

akuks commented on src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx:
```diff
@@ -17,12 +17,52 @@ const ManageSupportUI = () => {
+        <div className={styles.hrtag}></div>
         <div className={styles.notes}>
           {" "}
           Note : To change support order of camp, drag & drop the camp box on
```
Same as before

vikasranatalentelgia replied:
It is a static design implementation for now. will be dynamic once API will be implemented.

vikasranatalentelgia committed:
Merge branch 'development' into CAN-663

vikasranatalentelgia committed:
CAN-665, 713

Add Support: Implement Drag and Drop event on camps list
Error message displayed on "Change Password" page is wrong
